src/operators/fsApi/fsApi.ts

```diff
@@ -0,0 +1,42 @@
+import { Operator, OperatorOptions, OperatorValidator } from '../../operator';
+import { getGlobal } from '../../utils/object';
+
+export interface FSApiOperatorOptions extends OperatorOptions {
+
+}
+
+export default abstract class FSApiOperator implements Operator {
+  static specification = {};
+
+  constructor(public options:FSApiOperatorOptions) {
+    // sets this.options
+  }
+
+  handleData(data: any[]): any[] | null {
+    const thisArg: object = getGlobal();
+    // @ts-ignore
+    const fsFunction:any = thisArg[thisArg._fs_namespace]; // eslint-disable-line
```

Comment:
I wonder if there's a way to use the typings from `@fullstory/browser`. This way we get type safety from the official library.